docs/plugin_developer_guide.md

```diff
@@ -425,51 +427,57 @@ Use the guidelines below to compose command help.
   - Example: `bluemix test create (NAME | --uuid ID)`
 - "[...]" and "(...)" can be nested.
 - If a value accepts multiple type of inputs, it's recommended that for file type the file name should start with "@".
-- If a command has multiple paradigms and it's hard to describe them together, specify each of them in separate lines, 
+- If a command has multiple paradigms and it's hard to describe them together, specify each of them in separate lines,
   e.g.
   ```bash
   USAGE:
     bx test command foo.....
     bx test command bar.....
   ```
 
-The following gives an example of the out put of the "help" command:
+The following gives an example of the output of the `help` command:
```

Comment:
I tried to reword some of the example outputs for style/grammar because I noticed sometimes people have been copying these examples verbatim.